src/tcgwars/logic/impl/gen7/UltraPrism.groovy

```diff
@@ -3033,7 +3033,7 @@ public enum UltraPrism implements LogicCardInfo {
           text "You may play 4 Missing Clover cards at once. \n-If you played 1 card, look at the top card of your deck \n if you played 4 cards, take a prize card \nYou may play as many Item cards as you like during your turn (before your attack)."
           onPlay {
             if(my.hand.findAll({it.name=="Missing Clover"}).size()>=4) {
-              if(confirm("Use your 4 Missing Clover and take a prize card?")){
+              if(!my.deck || confirm("Use your 4 Missing Clover and take a prize card?")){
```

Comment:
Why is it important that your deck is not empty when you play 4 missing clovers?

Reply:
It should (in theory) avoid asking if you wanna play the 4 clovers together, since it should be the only option (you can't look at the top of an empty deck, so playing a single clover wouldn't be allowed then)